pints/_log_pdfs.py

```diff
@@ -59,16 +59,31 @@ class LogPrior(LogPDF):
 
     Extends :class:`LogPDF`.
     """
-    def sample(self, n=1):
+    def cdf(self, x):
         """
-        Returns ``n`` random samples from the underlying prior distribution.
+        Returns the cumulative density function at a point ``x``.
```

Comment:
put the dimensions required for x in the docstring. from the docstring it looks like x has to be a single point? Would it be possible to generalise this to allow multiple points, similar to the cdf and icdf functions in scipy? Same with docstring of other new functions below

Reply:
@martinjrobins Good idea! I have now tweaked the cdf and icdf functions to allow them to handle multiple points (most required no changes since they use the scipy functions that generalise this way automatically).

Reply:
thanks @ben18785. Can we also put something in the docstring indicating the shape of x that is required. I.e. ``x`` is an n x d array, where n in the number of input samples and d is the dimension of parameter space

Reply:
Good point -- done!